Add test setup and run instructions (#194)
sveltejs/sapper-template: README.md

Consult [sapper.svelte.dev](https://sapper.svelte.dev) for help getting started.

### Running tests

> Fonctional tests are located in `cypress` folder. Prior to running test, you'll need to install [cypress](https://www.cypress.io/) globally:

Reviewer: `Fonctional` should be changed to `Functional`. We also shouldn't be recommending a global install of cypress, but a local one.

Reviewer: Also `located in` should be `located in the` and `running test` changed to `running tests`.

> Functional tests are located in the [`cypress/integration`](cypress/integration) folder. Prior to running tests, you'll need to install [cypress](https://www.cypress.io/) globally:

Reviewer 2: ```suggestion
Functional tests are located in the [`cypress/integration`](cypress/integration) folder. Prior to running tests, you'll need to install [cypress](https://www.cypress.io/) as a dev dependency:
```

```bash

> npm i -g cypress

Reviewer 2: ```suggestion
npm install --save-dev cypress
```

```

Then you can run the tests using the following commands :
```bash

> npm t

Reviewer: ```suggestion
npm test
```

npm run cy:run

> npm run cy:open

Reviewer 2: These 3 lines can be combined to:

```bash
npm test
```

`test` [runs `cy:run`](https://github.com/sveltejs/sapper-template/blob/db19dc093ab38ff9a0ad234f444ce1683c96548d/package_template.json#L12).

`cy:open` opens the [interactive test runner](https://docs.cypress.io/guides/guides/command-line.html#cypress-open) which is not necessary. We may want to document it but I think that should be a new paragraph if we do.

> ```

Author: @antony, I've taken care of your request changes and added a link to the application specs.

Reviewer: :+1:  That still suggests a global install of cypress though. That needs to be fixed.

Reviewer 2: Made suggested changes for [dev dependency instead of global install](https://github.com/sveltejs/sapper-template/pull/194#discussion_r382356496) and [not abbreviated version of `npm install`](https://github.com/sveltejs/sapper-template/pull/194#discussion_r382357474) in separate review.

For `npm test` 3 lines can be combined I think so did not suggest change [in my comment](https://github.com/sveltejs/sapper-template/pull/194#discussion_r382359027) since those are limited to 1 line. 

Did address the  [`cy:run`](https://docs.cypress.io/guides/guides/command-line.html#cypress-run) (runs all tests headlessly in the Electron browser) vs [`cy:open`](https://docs.cypress.io/guides/guides/command-line.html#cypress-open) (opens the Cypress Test Runner in interactive mode). As I wrote there, we may want to document `cy:open` but I think that should be a new paragraph if we do.

> npm run cy:run

Reviewer: ```suggestion

```

> npm run cy:open

Reviewer: ```suggestion
```
